src/components/__tests__/EditVocabulary.test.tsx

```
@@ -0,0 +1,695 @@
+  };
+});
+vi.mock('@/lib/geminiUtils');
+
+// Mock other direct hook dependencies
+vi.mock('@/contexts/AuthContext');
+vi.mock('@/stores/languageStore');
+vi.mock('@/hooks/use-toast');
+vi.mock('@/lib/storyUtils');
+
+// Import after mocks
+import { supabase } from '@/integrations/supabase/client';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'supabase'. Consider to delete it or rename it.

'supabase' is defined here:


(lint/suspicious/noRedeclare)